solaris/solaris11/SPECS/template_agent_v3.9.4 copy.json

```diff
@@ -0,0 +1,1635 @@
+{
```

Comment:
Rename this file to `template_agent_v3.9.4.json`

Reply:
https://github.com/wazuh/wazuh-packages/pull/228/commits/2c323a91ed29fa10c8e1732e40b7f3665067c08e